src/MOIWrapper.jl

```diff
@@ -0,0 +1,561 @@
+end
+
+"""
+    change_objective_coefficient!(instance, col, coef)
+
+Set the linear coefficient of the variable in column `col` to `coef` in the objective function.
+"""
+function LQOI.change_objective_coefficient!(instance::ClpOptimizer, col, coef)
+    objcoefs = get_obj_coefficients(instance.inner)
+    objcoefs[col] = coef
+    chg_obj_coefficients(instance.inner, objcoefs)
+end
+
+"""
+    change_rhs_coefficient!(instance, row, coef)
+
+Set the rhs of the constraint in row `row` to `coef`.
+"""
+function LQOI.change_rhs_coefficient!(instance::ClpOptimizer, row, coef)
+    lower = replace_inf(get_row_lower(instance.inner))
+    upper = replace_inf(get_row_upper(instance.inner))
+    lb = lower[row]
+    ub = upper[row]
+    
+    if lb > -Inf
+        lower[row] = coef
+        chg_row_lower(instance.inner, lower)
+    elseif ub < Inf
+        upper[row] = coef
+        chg_row_upper(instance.inner, upper)
+    else
+        error("Either row_lower or row_upper must be of abs less than 1e20")
+    end
+end
+
+ 
+"""
+    delete_linear_constraints!(instance, start_row::Int, end_row::Int)::Void
+
+Delete the linear constraints `start_row`, `start_row+1`, ..., `end_row` from
+the model `m`.
+"""
+function LQOI.delete_linear_constraints!(instance::ClpOptimizer, start_row::Int, end_row::Int)::Void    
+    which = [Int32(i-1) for i in start_row:end_row]
+    delete_rows(instance.inner, which)
+end
+
+
+"""
+    change_linear_constraint_sense!(instance, rows::Vector{Int}, sense::Vector{Cchar})::Void
+
+Change the sense of the linear constraints in `rows` to `sense`.
+"""
+function LQOI.change_linear_constraint_sense!(instance::ClpOptimizer, rows::Vector{Int}, sense::Vector{Cchar} )::Void
+    lower = replace_inf(get_row_lower(instance.inner))
+    upper = replace_inf(get_row_upper(instance.inner))
+        
+    for (i,row) in enumerate(rows)                
+        lb = lower[row]
+        ub = upper[row]
+        if lb > -Inf
+            rhs = lb
+        elseif ub < Inf
+            rhs = ub
+        else
+            error("Either row_lower or row_upper must be of abs less than 1e20")
+        end
+        
+        sense = sense[i]
+        if sense == Cchar('G')
+            lb = rhs
+            ub = Inf
+        elseif sense == Cchar('L')
+            lb = -Inf
+            ub = rhs
+        elseif sense == Cchar('E')    
+            lb = rhs
+            ub = rhs
+        end
+        
+        lower[row] = lb
+        upper[row] = ub                
+    end
+    
+    chg_row_upper(instance.inner, upper)
+    chg_row_lower(instance.inner, lower)
+end
+
+
+
+"""
+    set_linear_objective!(instance, cols::Vector{Int}, coefs::Vector{Float64})::Void
+
+Set the linear component of the objective.
+"""
+function LQOI.set_linear_objective!(instance::ClpOptimizer, cols::Vector{Int}, coefs::Vector{Float64})::Void
+    obj_in = zeros(Float64, get_num_cols(instance.inner))
+    obj_in[cols] .= coefs
```

Comment:
Doesn't account for duplicates. See https://github.com/JuliaOpt/MathOptInterface.jl/pull/432